src/Init/Notation.lean

```diff
@@ -687,6 +687,11 @@ syntax (name := checkSimp) "#check_simp " term "~>" term : command
 -/
 syntax (name := checkSimpFailure) "#check_simp " term "!~>" : command
 
+/--
+#discr_tree_key t prints the discrimination tree key for `t`.
```

Comment:
It seems you command has two modes, worth documenting here. Maybe something like
```suggestion
#discr_tree_key t prints the discrimination tree key for the term `t`, or if `t`
is an identifier, for its type.
```